documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -29,7 +29,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
     git clone https://github.com/ReactionMechanismGenerator/RMG-Py.git
     git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
 
-#. Now create the anaconda environment for RMG-Py
+#. Now create the conda environment for RMG-Py
```

Comment:
The names of the environment files should also be updated below.

Reply:
Ah, sorry I forgot to make this change as well. I am pushing a fix-up commit with this change. Thanks for the catch!